### ISSUE TITLE
Adding user tags is impossible

### ISSUE THREAD

**Issue by mimmus** [mesosphere/konvoy-image-builder#464]
Our organization has some mandatory tag for EC2 instances, creating also a temporary instance without tags is impossible.
Looking at Packer template, adding custom tags is actually impossible.

**Comment by faiq** [comment]
Adding user tags is not impossible, you can make customizations to the packer manifest that kib produces and use it for your builds.

For example if I need to create an image with with custom tags 

```json 
{
"my_cool": "tag"
}
``` 

I would do it by running the following steps

1. Generate manifest files for the KIB run by doing 

```
$ ./konvoy-image build images/ami/centos-79.yaml  
e2022/08/11 19:01:17 writing new packer configuration to work/centos-7-1660244477-JpGye
2022/08/11 19:01:17 starting packer build
centos-7.9: output will be in this color.
``` 

and sending `SIGINT` to the process to kill it 

2. Next edit the file `packer.json` from the above work directory. In my case the path was `work/centos-7-1660244477-JpGye/packer.json` 

3. When editing go to builders section of the json and add a [run_tags](https://www.packer.io/plugins/builders/amazon/ebs#run_tags) block. When looking at it with jq i get the following result: 

```
$ cat work/centos-7-1660244477-JpGye/packer.json | jq '.builders|.[]|."run_tags"'
{
  "my_cool": "tag"
}
``` 

4. Pass the modified packer file to konvoy-image like so:

`$ ./konvoy-image build --packer-manifest=work/centos-7-1660244477-JpGye/packer.json images/ami/centos-79.yaml` 

You will see the logs 

```
    centos-7.9: Found Image ID: ami-0686851c4e7b1a8e1
==> centos-7.9: Creating temporary keypair: packer_62f554df-e60d-8919-4b0e-3c9ba405551c
==> centos-7.9: Creating temporary security group for this instance: packer_62f554e6-3004-1d78-1454-fa28bb3da4e7
==> centos-7.9: Authorizing access to port 22 from [0.0.0.0/0] in the temporary security groups...
==> centos-7.9: Launching a source AWS instance...
    centos-7.9: Adding tag: "my_cool": "tag"
    centos-7.9: Instance ID: i-0be42f1c3f6455ad7
``` 

And the instance will be created with the tag you need. 


We will be adding a feature to pass this in via command line, but until then this is how you can do it.

**Comment by mimmus** [comment]
This workaround is really useful.
Thanks

**Comment by faiq** [comment]
@mimmus This will be included in the next release 

you can specify 

```
packer:
  run_tags:
    my_cool:tag
``` 

in your image.yaml file.

**Comment by faiq** [comment]
If you need to open another issue please let us know!

**Comment by mimmus** [comment]
But by "next release", do you mean "major release" or "minor release" for every supported DKP version?

**Comment by faiq** [comment]
its available in konvoy-image-builder v2.0.0 which will be used by dkp 2.5.0
